portal-web/test/functional/com/liferay/portalweb/macros/Form.macro

```diff
@@ -35,9 +35,17 @@ definition {
 	}
 
 	macro addTextField {
-		Form.gotoAddField(
-			fieldType = "Text",
-			key_fieldPositionNumber = "${fieldPositionNumber}");
+		if (isSet(formPageNumber)) {
+			Form.gotoAddField(
+				fieldPositionNumber = "${fieldPositionNumber}",
+				fieldType = "Text",
+				formPageNumber = "${formPageNumber}");
```

Comment:
@mlan0, can you please sort alphabetically?

Reply:
It's already sorted.